libpod/container_internal.go

```diff
@@ -350,7 +350,7 @@ func (c *Container) setupStorage(ctx context.Context) error {
 		},
 		LabelOpts: c.config.LabelOpts,
 	}
-	if c.config.Privileged {
+	if c.config.Privileged && c.runtime.config.StorageConfig.GraphDriverName != "vfs" {
```

Comment:
There's an awful lot of magicness here. Future maintainers might appreciate a comment explaining why this is disabled for vfs; or, perhaps even better, a helpfully-named one-shot function such as GraphDriverSupportsPrivilegedOptions() or whatever; something that can be extended as/if needed some day if the problem ever affects more than just vfs.

A comment might be welcome one line down, too, explaining just what is special about those three options.